core/src/main/java/com/linecorp/armeria/internal/common/stream/SubscriptionArbiter.java

```diff
@@ -191,18 +176,18 @@ final void drain() {
 
         do {
             // Get snapshots from volatile values and initialize them
-            long newReq = newRequested;
+            final long newReq = newRequested;
             if (newReq != 0L) {
```

Comment:
How about removing this if branch because assigning 0 is cheaper than having this branch?